### DISCUSSION TITLE
Ruby 3.0.3 released, with some fix for thread/ fiber scheduler, so, how we start ruby concurrency on that?

### DISCUSSION THREAD

**Discussion by zw963** [ioquatix/ioquatix#8] [General]
Hi, Ruby 3.0.3 released yesterday, commit logs is [here](https://github.com/ruby/ruby/compare/v3_0_2...v3_0_3)

i remember i saw one video from you, which said, Ruby 3.0.3 including some bug fix for it.

I am a new guy to fiber scheduler, i am exciting to know all of the latest progress about the gems, versions, features which
production ready for Ruby 3.0.3, in fact, i want know every basic things which can make ruby programming concurrency easy,
and the most part, performance improvement, summarize all of gems, apis which we should be concerned.

sorry for bad english,  i hope you can understand what i means.

Thank you!

**Comment by ioquatix** [comment]
I will spend some time on this on the weekend. All the bug fixes we needed are included which is good news.
